app/services/card_comparator.rb

```diff
@@ -10,18 +10,17 @@ def initialize(params)
   end
 
   def self.call(params)
-  	new(params).right?
+  	new(params).diff
   end
 
-  def right?
-  	if @card.original_text.downcase.strip == @compared_text.downcase.strip  # is the answer right?
-      update_card_attr_right!(@card)   # yes, so update correct, reser wrong counters, return true
-      return true
+  def diff
+  	difference = DamerauLevenshtein.distance(@card.original_text.downcase.strip, @compared_text.downcase.strip, 1, 2)
```

Comment:
Line is too long. [116/80]<br>Use 2 (not 1) spaces for indentation.<br>Tab detected.